### DISCUSSION TITLE
Support glob pattern in uses()->in()

### DISCUSSION THREAD

**Discussion by walirazzaq** [pestphp/pest#481] [Ideas]
While building a modular laravel applications where modules contains their own tests. Currently we have to do something like following to make it work
```PHP
   uses(Tests\TestCase::class)->in(
        'Feature',
        '../app-modules/product/tests/Feature',
        '../app-modules/order/tests/Feature',
        '../app-modules/blog/tests/Feature',
    )
```
List could go long so.
I see a [PR](https://github.com/pestphp/pest/pull/312) regarding support of `Finder` but it couldn't make it to happen owing to some reason.
I am experimenting with `glob()` along with `is_dir()` & `file_exists()`
https://github.com/pestphp/pest/blob/04663e0c8e046fafaa7006ea9f6803623e1833e0/src/PendingCalls/UsesCall.php#L79

Like so
```PHP
        $this->targets = array_reduce($targets, function (array $accumulator, string $target): array {
            foreach (glob($target) as $path) {
                if (is_dir($path) || file_exists($path)) {
                    $accumulator[] = (string) realpath($path);
                }
            }

            return $accumulator;
        }, []);
```
This solved the issues of using basic glob patterns supported by [glob()](https://www.php.net/manual/en/function.glob.php)
```PHP
   uses(Tests\TestCase::class)->in(
        'Feature',
        '../app-modules/*/tests/Feature',
        '../app-modules/*/tests/Unit/*SomeTest.php',
    )
```
Thanks for your time and making us to love testing ♥️

**Comment by nunomaduro** [comment]
cc @owenvoke
